imagemounter/volume_system.py

```diff
@@ -279,6 +305,11 @@ def _detect_mmls_volumes(self, vstype='detect'):
                 except Exception as e:
                     logger.exception("Failed executing mmls command")
                     raise SubsystemError(e)
+            # since mmls cannot determine the partition type for exFAT superfloppy formatting; check for this.
+            elif self._detect_exfat_superfloppy():
```

Comment:
With these details, I can reproduce this issue:

```
# dd if=/dev/zero of=exfat_test.dd bs=1M count=1
# mkfs.exfat -n '8GB_exFAT' exfat_test.dd
# parted exfat_test.dd print free
Model:  (file)
Disk exfat_test.dd: 1049kB
Sector size (logical/physical): 512B/512B
Partition Table: msdos
Disk Flags: 

Number  Start   End     Size    Type  File system  Flags
        16.4kB  1049kB  1032kB        Free Space
```

The thing is, this seems to be a bug in `parted` detecting 'something' (i.e., a DOS partition) but not everything. Basically, we need to tell imagemounter that it did not detect anything. That would then result in it mounting as a single volume (which it should). 

I'm unsure whether we should keep this as a quirk that we should not fix (other detection methods work fine, as well as `--single`) or as something we should fix. And secondly, how we should fix it. I am sure that I do not want to check for the specific case of EXFAT here. I am also sure that we should leave detection as a single volume to the `detect_volumes` method.

Perhaps we could detect whether the msdos detection makes sense? It only detects a free volume far away from the start of the disk, which is quite strange...